doc/user_guide/impala_user_guide.md

```diff
@@ -25,7 +25,9 @@ You need to specify the following settings when adding the JDBC driver via EXAOp
 | Prefix    | `jdbc:impala:`                                          |
 | Files     | `ImpalaJDBC41.jar`                                      |
 
-## Uploading the JDBC Driver to EXAOperation
+IMPORTANT: Currently you have to Disable Security Manager for the driver if you want to connect to Impala using Virtual Schemas. It is necessary because JDBC driver requires Java permissions which we do not grant by default.
```

Comment:
```suggestion
IMPORTANT: Currently you have to [Disable Security Manager](https://docs.exasol.com/administration/on-premise/manage_software/manage_jdbc.htm) for the driver if you want to connect to Impala using Virtual Schemas. It is necessary because JDBC driver requires Java permissions which we do not grant by default.
```

Reply:
Please check if this link makes sense. I couldn't find any other doc for security manager, update it if there is dedicated page.

Reply:
I think this link is fine, I also can't find anything that would contain more info